src/stores/Frame/FrameStore.ts

```diff
@@ -102,7 +102,7 @@ export class FrameStore {
     private static readonly ZoomInertiaDuration = 250;
     private static readonly CursorMovementDuration = 250;
 
-    private spectralFrame: AST.SpecFrame;
+    public spectralFrame: AST.SpecFrame;
```

Comment:
@crocka Please change it back to private.
Please also update the changelog, thanks.